src/GitHub.InlineReviews/Margins/InlineCommentMarginEnabled.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Utilities;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace GitHub.InlineReviews.Margins
+{
+    [Export(typeof(EditorOptionDefinition))]
+    [Name(OptionName)]
+    public class InlineCommentMarginEnabled : ViewOptionDefinition<bool>
```

Comment:
I've never used this API, so I'm missing something here, but the only thing that seems to be used from this class is `OptionName`. Do we need to create a subclass of `ViewOptionDefinition` if we only use the name? Why not use an instance of this class as they key when getting options?

Reply:
By exporting `ViewOptionDefinition` we declare than a new options exists on the text views (and give it a default value). I probably shouldn't be using `OptionName` directly and should be importing the option and using the `Key` property. This gives us type information for the property.